app/views/passengers/brochure.html.haml

```diff
@@ -1,76 +1,75 @@
-%h1 SpecTrans Passenger Information
-.row
-  .col-md-6
-    %table.table
-      %caption Contact Information
-      %thead
-      %tbody
-        %tr
-          %th Phone
-          %td 413-545-2086
-        %tr
-          %th Public Website
-          %td= link_to 'www.umass.edu/transportation/special-transportation',
-            'https://www.umass.edu/transportation/special-transportation'
-        %tr
-          %th Email
-          %td stspv@umass.edu
-.row
-  .col-md-6
-    %table.table
-      %caption Special Transportation Staff
-      %thead
-      %tbody
-        %tr
-          %th Supervisors
-          %td Lisa Gagnon &amp; Shayla Keane
-        %tr
-          %th Coordinator
-          %td Diana Noble
-%h2 Hours of Operation
-.row
-  .col-md-6
-    %table.table
-      %caption Semester
-      %thead
-        %tr
-          %th{ 'aria-label': 'Days' }
-          %th Dispatch
-          %th Van
-      %tbody
-        %tr
-          %th{ scope: 'row'} Monday Through Friday
-          %td 7:00 AM to 5:00 PM
-          %td 7:15 AM to 8:15 PM
-        %tr
-          %th{ scope: 'row' } Saturday
-          %td none
-          %td 10:00 AM to 8:00 PM
-        %tr
-          %th{ scope: 'row' } Sunday
-          %td none
-          %td 10:00 AM to 8:00 PM
-.row
-  .col-md-6
-    %table.table
-      %caption Reduced Service (Holidays, Summer, Intersession)
-      %thead
-        %tr
-          %th{ 'aria-label': 'Days' }
-          %th Dispatch
-          %th Van
-      %tbody
-        %tr
-          %th{ scope: 'row'} Monday Through Friday
-          %td 8:00 AM to 5:00 PM
-          %td 7:15 AM to 7:15 PM
-        %tr
-          %th{ scope: 'row' } Saturday
-          %td none
-          %td 11:15 AM to 4:45 PM
-        %tr
-          %th{ scope: 'row' } Sunday
-          %td closed
-          %td closed
-= render 'policies'
-= render 'scheduling_rides_information'
+#brochure.container
+  %h1 SpecTrans Passenger Information
+  .row
+    .col-lg-6
+      %table.table
+        %caption Contact Information
+        %thead
+        %tbody
+          %tr
+            %th Phone
+            %td 413-545-2086
+          %tr
+            %th Public Website
+            %td= link_to 'www.umass.edu/transportation/special-transportation',
+              'https://www.umass.edu/transportation/special-transportation'
+          %tr
+            %th Email
+            %td stspv@umass.edu
+    .col-lg-6
+      %table.table
+        %caption Special Transportation Staff
+        %thead
+        %tbody
+          %tr
+            %th Supervisors
+            %td Lisa Gagnon &amp; Shayla Keane
+          %tr
+            %th Coordinator
+            %td Diana Noble
+  %h2 Hours of Operation
+  .row
+    .col-lg-6
+      %table.table
+        %caption Semester
+        %thead
+          %tr
+            %th{ 'aria-label': 'Days' }
```

Comment:
Again, not necessarily layout, more just a curiosity, but why not just make this normal header text? seems odd to me.

Reply:
I don't remember, but I'm pretty sure this page was initially constructed by copying from a Word document, so it was a decision made by some staff member, copied to Haml by @Anbranin, and then she decided to add a little bit of A11y help without changing the formatting.

Practically speaking, I think the content in this column is pretty evident... once you get to it. But this extra hint is possibly helpful for screen-reader users?

Reply:
Yeah makes sense to me, the only reason I actually noticed it was because the tables look a little funny with no header on the left column. But it doesn't really matter